doma-spring-boot-core/src/main/java/org/seasar/doma/boot/DomaPersistenceExceptionTranslator.java

```diff
@@ -63,7 +78,8 @@ public DataAccessException translateExceptionIfPossible(RuntimeException ex) {
 			if (ex instanceof SqlExecutionException) {
 				sql = ((SqlExecutionException) ex).getRawSql();
 			}
-			return translator.translate(ex.getMessage(), sql, e);
+			DataAccessException dae = translator.translate(ex.getMessage(), sql, e);
+			return (dae != null ? dae : new UncategorizedSQLException("", sql, e));
```

Comment:
I think better that set `ex.getMessage()` instead of `""` at first argument of `UncategorizedSQLException` for backwards compatibility. WDYT?

*  https://github.com/spring-projects/spring-framework/blob/v5.2.11.RELEASE/spring-jdbc/src/main/java/org/springframework/jdbc/support/AbstractFallbackSQLExceptionTranslator.java#L89

Reply:
Thanks!

I thought about the same thing and fixed it. 66705cb4a3d9449e45135a31254aff323adbe987